site/pages/docs/versions/v4.0/v4.0.31-en.hbs

```diff
@@ -0,0 +1,163 @@
+---
+{
+	"title": "v4.0.31 release notes",
+	"language": "en",
+	"description": "v4.0.31 release notes - Version history - Web Experience Toolkit (WET)",
+	"altLangPrefix": "v4.0.31",
+	"dateModified": "2019-06-19"
+}
+---
+<p>Version 4.0.31</p>
+<ul>
+	<li><strong>Launching date:</strong>
+		<time>2019-06-20</time>
+	</li>
+	<li><a href="../dwnld-en.html#v4031">Downloads</a></li>
+</ul>
+<div class="wb-prettify all-pre"></div>
+<section>
+	<h2>What’s new?</h2>
+	<ul>
+		<li>Template
+			<ul>
+				<li>Minor - Added <code>nav</code> element to skip links, as <a href="#nav-ex">shown here</a></li>
+			</ul>
+		</li>
+		<li>Styles and plugins
+			<ul>
+				<li>Minor - Overlay - Added optional background for overlays with <code>.overlay-bg</code> class and added an "opened" and "closed" event</li>
+				<li>Minor - Share widget - Removed deprecated social sites in the interface (Google+, Delicious and StumbleUpon)</li>
+				<li>Minor - Share widget - Updated Twitter share link to match the new pattern (<a href="#twitter-ex">see what has changed</a>)</li>
+				<li>Minor - Geomap - Added filtering with URL manager compatibility</li>
+				<li>Minor - Geomap - Migrated Openlayer from version 2.x to 3.x (<a href="https://github.com/wet-boew/wet-boew/pull/8578">more information</a>)</li>
+				<li>Minor - Core - Added a download function that uses a blob to the core functions</li>
+				<li>Minor - Tables - Added advanced options for filtering (<a href="https://wet-boew.github.io/wet-boew/docs/ref/tables/tables-en.html#advopt">read documentation to learn more</a>)</li>
+				<li>Patch - Tables - Fixed checkbox filtering when followed by a <code>&lt;select&gt;</code> element.</li>
+				<li>Patch - Tables - Removed text duplication upon initialization</li>
+				<li>Patch - Multimedia player - Fixed player playing in the background when contained in an overlay.</li>
+				<li>Patch - Feeds - Fixed condition when limit is greater than the number of entries</li>
+				<li>Patch - Toggle - Ajusted <code>&lt;detail&gt;</code> toggle behaviour to better match natural cycle when there is multiple details to toggle at once</li>
+			</ul>
+		</li>
+		<li>Maintenance
+			<ul>
+				<li>Patch - Fixed blocking condition in the build script related to sprites</li>
+			</ul>
+		</li>
+	</ul>
+	<section>
+		<h3>Browser supported:</h3>
+		<p>As per the rule defined by <a href="http://wet-boew.github.io/wet-boew-documentation/decision/2.html">Design decision 2: Browser supported</a></p>
+		<ul>
+			<li>Chrome 71</li>
```

Comment:
The browser supported list need to be updated as per the [design decision #2](https://wet-boew.github.io/wet-boew-documentation/decision/2.html)
Like Chrome version should be 74 & 75